paper/writeup.tex

```diff
@@ -1203,11 +1236,12 @@ \section{Discussion}
 Clearly, there is ample information in genomic data to infer the extent of ancestral haplotypes even where they are unary,
 and this has the potential to significantly inform downstream methods, especially dating.
 However, currently not all ARG inference methods return ARGs with unary nodes.
-For instance, we have seen that tsinfer returns ARGs to which the method presented here adds little -
+For instance, we have seen that tsinfer returns ARGs to which the method presented here adds little --
 so, seems to be already taking advantage of this source of information.
-Singer [cite] returns ARGs without any unary nodes,
-although these are reconstructable using the times of the recombination nodes.
-On the other hand, locally building trees separately in each segment of genome
```

Comment:
This was based on a conversation wtih Yun Deng on slack - so, it'd be nice to insert it, unless we think it's wrong? (And, it would be good to cite ARGNeedle somewhere, ideally earlier...)

Reply:
As far as I know, SINGER doesn't return recombination nodes either, so this seemed like a confusing and unnecessary level of detail. (But, maybe this has changed in a recent version of SINGER?) If you think it's important, I'm fine w/ adding it back it

Reply:
I'd be really surprised if SINGER doesn't return recombination nodes - that's the model it's inferring after all.

Reply:
It doesn't stick them into the output tree sequence, only coalescing nodes (at least last time I looked, a couple months ago). They may be stored in its own internal format, but there's no documentation on this as far as I know.